record/assets/views/record.html

```diff
@@ -38,7 +38,7 @@
                                 {{value | sanitizeValue }}
                         </a>
                         <div ng-class="{'fixed-font' : entity.sequences.indexOf(key) > -1, '' : entity.sequences.indexOf(key) == -1}">
-                            <span ng-if="!entity[key + '_link'] || value === null" ng-bind-html="value | sanitizeValue"></span>
+                            <span ng-if="!entity[key + '_link'] || value === null" ng-bind-html="value | sanitizeValue" ng-class="entity.colDataTypes[key]=='markdown'?'markdown-container':''"></span>
```

Comment:
This is more a style question/preference, but why not use: 
`ng-class="'markdown-container': entity.colDataTypes[key]=='markdown'"`? I find it easier to read and you don't need the "else" branch this way.